mne/io/egi/egimff.py

```diff
@@ -686,3 +692,187 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
 
         # do the calibration
         _mult_cal_one(data, one, idx, cals, mult)
+
+
+@verbose
+def read_evokeds_mff(fname, condition=None, channel_naming='E%d',
+                     baseline=None, verbose=None):
+    """Read averaged MFF file as `EvokedArray` or list of `EvokedArray`.
+
+    Parameters
+    ----------
+    fname : str
+        File path to averaged MFF file. Should end in .mff.
+    condition : int or str | list of int or str | None
+        The index (indices) or category (categories) from which to read in
+        data. Averaged MFF files can contain separate averages for different
+        categories. These can be indexed by the block number or the category
+        name. If `condition` is a list or None, a list of Evoked objects is
+        returned.
+    channel_naming : str
+        Channel naming convention for EEG channels. Defaults to 'E%%d'
+        (resulting in channel names 'E1', 'E2', 'E3'...).
+    baseline : None (default) or tuple of length 2
+        The time interval to apply baseline correction. If None do not apply
+        it. If baseline is (a, b) the interval is between "a (s)" and "b (s)".
+        If a is None the beginning of the data is used and if b is None then b
+        is set to the end of the interval. If baseline is equal to (None, None)
+        all the time interval is used. Correction is applied by computing mean
+        of the baseline period and subtracting it from the data. The baseline
+        (a, b) includes both endpoints, i.e. all timepoints t such that
+        a <= t <= b.
+    %(verbose)s
+
+    Returns
+    -------
+    evoked : EvokedArray or list of EvokedArray
+        The evoked dataset(s); one EvokedArray if condition is int or str,
+        or list of EvokedArray if condition is None or list.
+
+    Raises
+    ------
+    ValueError
+        If `fname` has file extension other than '.mff'.
```

Comment:
This tries to link but it can't (NumpyDoc/autodoc do not make the params a target), leading to Sphinx emitting these warnings that get treated as errors:
```
/home/circleci/project/mne/io/egi/egimff.py:docstring of mne.read_evokeds_mff:11: WARNING: py:obj reference target not found: condition
/home/circleci/project/mne/io/egi/egimff.py:docstring of mne.read_evokeds_mff:48: WARNING: py:obj reference target not found: fname
/home/circleci/project/mne/io/egi/egimff.py:docstring of mne.read_evokeds_mff:51: WARNING: py:obj reference target not found: fname
/home/circleci/project/mne/io/egi/egimff.py:docstring of mne.read_evokeds_mff:54: WARNING: py:obj reference target not found: fname
```
I would make these double-backticks so that they get rendered as code